client/src/components/Download.vue

```diff
@@ -95,8 +95,17 @@ export default {
       this.loading = false;
     },
     async download_plotImage() {
-      await this.$store.createPlot();
```

Comment:
Can you also remove all the stuff that references the API for createPlot()?
We dont need them anymore
I couldnt do it in my branch because they'd break the front end. 
Remember to merge dev first.

Reply:
Yes, sorry I just saw this now! Fixed.